### ISSUE TITLE
[Items] 'X Added Skill is a Jewel Socket' is not shown upon price checking

### ISSUE THREAD

**Issue by gwarlek** [domialex/Sidekick#418]
Sample item. The affix is rather important

```
Rarity: Rare
Oblivion Heart
Medium Cluster Jewel
--------
Item Level: 70
--------
Adds 5 Passive Skills (enchant)
Added Small Passive Skills grant: +4% to Chaos Damage over Time Multiplier (enchant)
--------
1 Added Passive Skill is Brewed for Potency
1 Added Passive Skill is Circling Oblivion
1 Added Passive Skill is a Jewel Socket
--------
Place into an allocated Medium or Large Jewel Socket on the Passive Skill Tree. Added passives do not interact with jewel radiuses. Right click to remove from the Socket.
--------
```

![Oblivion Heart](https://i.imgur.com/tYmnwQg.png)

**Comment by leMicin** [comment]
This is a hard one to solve.

The API sends this line of text
- # Added Passive Skills are Jewel Sockets

We need to detect this line from the game
- 1 Added Passive Skill is a Jewel Socket

We would really like a fix that does not involve hard coding singular / plural forms of a modifier.

**Comment by leMicin** [comment]
Fixed in 2.0
